thoth-api/src/series/model.rs

```diff
@@ -25,12 +27,20 @@ pub enum SeriesType {
     derive(juniper::GraphQLEnum),
     graphql(description = "Field to use when sorting series list")
 )]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, EnumString, Display)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
 pub enum SeriesField {
+    #[serde(rename = "SERIES_ID")]
+    #[strum(serialize = "ID")]
     SeriesID,
     SeriesType,
+    #[strum(serialize = "Series")]
     SeriesName,
+    #[serde(rename = "ISSNPRINT")]
```

Comment:
I hadn't realised these were the enum values we had for ISSN. It'd make more sense to have them as `ISSN_PRINT` and `ISSN_DIGITAL`

Reply:
Agreed - couldn't remember how to change this when I first encountered it, but I've sorted it now :)